athletics.lic

```diff
@@ -0,0 +1,170 @@
+custom_require.call(%w(common common-travel))
+
+class Athletics
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    @settings = get_settings()
+    @athletics_options = get_data('athletics').athletics_options
+
+    @climbing_target = get_data('athletics').practice_options[@settings.climbing_target]
+    
+    @start_skill=DRSkill.getxp('Athletics')
+    @end_skill=@start_skill+15
+    if(@end_skill>29)
+      @end_skill=29
+    end
+
+    if @climbing_target
+      walk_to(@climbing_target['id'])
+      climb_practice(@climbing_target['target'], @climbing_target['hide'])
+    elsif @settings.hometown == 'Crossing' || (@settings.hometown == 'Riverhaven' and DRSkill.getrank('Athletics') > 140)
+      crossing_athletics
+    elsif @settings.hometown == 'Shard'
+      shard_athletics
+    elsif @settings.hometown == 'Hibarnhvidar'
+      hib_athletics
+    elsif @settings.hometown == 'Ratha'
+      ratha_athletics
+    end
+  end
+  
+  def done_training?
+    return DRSkill.getxp('Athletics') >= @end_skill
```

Comment:
You will need to add `drinfomon` to your `custom_require` list to use DRSkill and DRRoom